games/devtest/mods/testtranslations/init.lua

```diff
@@ -0,0 +1,17 @@
+local S, NS = minetest.get_translator("testtranslations")
+
+minetest.register_chatcommand("testtranslations", {
+	params = "",
+	description = "Test translations",
+	privs = {},
+	func = function(name, param)
+		minetest.chat_send_player(name, "Please ensure your locale is set to \"fr\"")
```

Comment:
```suggestion
		if minetest.get_player_information(name).language ~= "fr" then
				return false, "Please restart Minetest with language = fr"
		end

```

Reply:
I applied the suggestion, but I must have forgotten to pull before rebasing, deleting the commit. I'll commit it again, thanks for catching it!
Although, now that I'm thinking about it, we really want to test this both with `fr` and default locale (to see both untranslated and translated strings), I'll need to check what `lang_code` is in that case.